res/controllers/Behringer-Extension-scripts.js

```diff
@@ -467,13 +502,13 @@
      */
     var BackLoopButton = function(options) {
         options = options || {};
-        options.outKey = options.outKey || "loop_enabled";
+        options.key = options.key || "loop_enabled";
         components.Button.call(this, options);
     };
     BackLoopButton.prototype = deriveFrom(components.Button, {
-        input: function(_channel, _control, value, _status, group) {
-            var engine = global.engine;
+        inSetValue: function(value) {
             var script = global.script;
+            var group = this.group;
```

Comment:
why is this line required?

Reply:
This is required because `group` is no longer an argument if the surrounding function. `group` is used 7 times within the function, so it either has to be declared like this or I'd have to change the 7 occurrences to `this.group`.

Reply:
Thanks, makes sense. I guess thats valid.